optimize/backend/pom.xml

```diff
@@ -213,7 +213,7 @@
     <dependency>
       <groupId>com.opencsv</groupId>
       <artifactId>opencsv</artifactId>
-      <version>5.9</version>
+      <version>5.10</version>
```

Comment:
We should remove this explicit version from here and define it in `parent/pom.xml` in the `properties` section